department-of-veterans-affairs/va.gov-cms
# VACMS-19496: Enable content sync with settings.

## Description
This work enables the content sync module and pushes the initial settings file.

Relates to #19496 

## Testing done
Local testing

## QA steps

What needs to be checked to prove this works?
Site behavior shouldn't change at all.
What needs to be checked to prove it didn't break any related things?
We should be able to import and export content/users.
What variations of circumstances (users, actions, values) need to be checked?
Using drush in a terminal should be possible to import/export.

### Definition of Done


### Select Team for PR review
